tests/test_views.py

```diff
@@ -93,16 +93,20 @@ def test_get_workflows_missing_user(app):
 
 
 def test_create_workflow_with_name(
-    app, session, default_user, tmp_shared_volume_path, cwl_workflow_with_name
+    app, session, default_user, cwl_workflow_with_name, tmp_shared_volume_path
 ):
     """Test create workflow and its workspace by specifying a name."""
     with app.test_client() as client:
         res = client.post(
             url_for("workflows.create_workflow"),
-            query_string={"user": default_user.id_},
+            query_string={
+                "user": default_user.id_,
+                "workspace_root_path": tmp_shared_volume_path,
+            },
             content_type="application/json",
             data=json.dumps(cwl_workflow_with_name),
         )
+        print(res.get_data(as_text=True))
```

Comment:
Very minor: I guess we can remove this print statement as well as the one in line 267